shared/common-adapters/avatar.js

```diff
@@ -223,7 +224,7 @@ class Avatar extends Component<void, Props, State> {
         onClick={this.props.onClick}
         opacity={this.props.opacity}
         size={this.props.size}
-        style={this.props.style}
+        style={{...this.props.style, ...(this.props.onClick ? globalStyles.clickable : null)}}
```

Comment:
sadly this doesn't work in RN. you have to pass {} instead of null. they have some non-spec implementation of object rest spread (https://github.com/facebook/react-native/blob/7a4eda2f70aea5bcbd46d295b873788a100c18bb/Libraries/polyfills/Object.es6.js#L17)